### ISSUE TITLE
"Confirm merge" message requirement

### ISSUE THREAD

**Issue by SurmaAa** [gitbucket/gitbucket#2530]
Hi, Is it possible to set `confirm merge message` textarea as required?
![Screenshot from 2020-08-28 14-05-36](https://user-images.githubusercontent.com/40576160/91559006-92960600-e937-11ea-9a2d-cc8184c7ba2b.png)

**Comment by SIkebe** [comment]
@SurmaAa Could you elaborate? The textarea is already marked as required.

![image](https://user-images.githubusercontent.com/17608272/91631114-2a682280-ea12-11ea-97fc-9702c7b223e1.png)

**Comment by SurmaAa** [comment]
@SIkebe Yes, but by default the textarea value is the PR branch name so it will let me confirm merge even if did not edited it.

**Comment by ctrl-chris-h** [comment]
@SIkebe I think @SurmaAa is talking about how GitBucket throws in a default message for you (ultimately meaning you don't have to put anything there). Is there a way to change this default message maybe?

**Comment by SurmaAa** [comment]
@ctrl-chris-h that's exactly what was on my mind
